Web/Edubase.Services.TexunaUnitTests/Lookup/LookupApiServiceTests.cs

```diff
@@ -67,16 +67,6 @@ public async Task EstablishmentTypesGetAllAsyncTest()
             Assert.Equal(_lookupDtoList.Count, result.Count());
         }
 
-        [Theory]
-        [MemberData(nameof(GetSyncronousLookupTypes))]
-        public void SyncronousLookupsTest(string lookupType)
-        {
-            //use reflection to construct method name
-            var method = _lookupApiService.GetType().GetMethod($"{lookupType}GetAll");
-            var ex = Assert.Throws<System.Reflection.TargetInvocationException>(() => method.Invoke(_lookupApiService, null));
-            Assert.IsType<NotImplementedException>(ex.InnerException);
-        }
-
```

Comment:
Note to self -- this test is removed because the synchronous (`NotImplementedException`) methods were removed (commented out) from `LookupApiService.cs`